docs/index.rst

```diff
@@ -15,6 +15,7 @@ Welcome to FAIR Universe HEP Challenge's documentation!
    pages/evaluation.md
    pages/prizes.md
    pages/starting_kit.md
+   pages/advanced_example.md
```

Comment:
I am thinking it moving it closer to the code. 
I will do it in another PR do for now this it good.